index.md

```diff
@@ -17,7 +17,7 @@ Below is a list of projects which are continually being added to. We welcome pro
 - [NHS GreenSpace](https://nhsx.github.io/nhsx-internship-projects/nhs-greenspace/)
 - [NHS Language Corpus](https://nhsx.github.io/nhsx-internship-projects/nhs-language-corpus/)
 - [NHS Semantic Search](https://nhsx.github.io/nhsx-internship-projects/nhs-semantic-search/)
-- [Predicting outcomes using SAIL Databank](https://nhsx.github.io/nhsx-internship-projects/sail-databank-outcomes-prediciton/)
+- [Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank](https://nhsx.github.io/nhsx-internship-projects/Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/)
```

Comment:
Where possible I shortened the slugs, so have done this here as well.

`/Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/`
-> `/transforming-healthcare-data-graph-based-sail/`

Reply:
changed